resources/js/Molecules/FormRadioButton.tsx

```diff
@@ -6,14 +6,16 @@ interface choice {
 	value: string;
 }
 interface IProps {
+	type:string;
```

Comment:
It seems like only certain strings are valid, not any old string - is that correct? If so, I think it would be a good idea to use a [union type](https://www.typescriptlang.org/docs/handbook/advanced-types.html#union-types) here, listing the only valid values like this:
```suggestion
	type: "checkbox" | "radio";
```